Domain Connect Spec Draft.adoc

```diff
@@ -309,45 +309,50 @@ Provider. This JSON structure will contain the following fields.
 |*Key*
 |*Type*
 |*Description*
-|Provider Id
+|*Provider Id*
 |providerId
 |String
 |Unique identifier for the DNS Provider. Typically, the domain name (e.g. virtucom.com).
 
-|Provider Name 
-|providerName 
+|*Provider Name* 
+|providerName
 |String 
 |The name of the DNS Provider.
 
-|Provider Display Name 
+|*Provider Display Name* 
 |providerDisplayName 
 |String 
 |The name of the DNS Provider that should be displayed by the Service Provider. Note: This might change for some DNS Providers that white label their infrastructure.
 
-|UX URL Prefix for Synchronous Flows 
+|*UX URL Prefix for Synchronous Flows* 
 |urlSyncUX 
 |String 
 |The URL Prefix for linking to the UX of Domain Connect for the synchronous flow at the DNS Provider. If not returned, the DNS Provider is not supporting the synchronous flow on this domain.
 
-|UX URL Prefix for Asynchronous Flows 
+|*UX URL Prefix for Asynchronous Flows*
 |urlAsyncUX 
 |String 
 |The URL Prefix for linking to the UX elements of Domain Connect for the asynchronous flow at the DNS Provider. If not returned, the DNS Provider is not supporting the asynchronous flow on this domain.
 
-|API URL Prefix 
+|*API URL Prefix* 
 |urlAPI 
 |String 
 |This is the URL Prefix for the REST API
 
-|Width of Window 
+|*Width of Window*
 |width 
 |Number 
 |This is the desired width of the window for granting consent when navigated in a popup. Default value is 750px.
 
-|Height of Window 
+|*Height of Window* 
 |height 
 |Number 
 |This is the desired height of the window for granting consent when navigated in a popup. Default value is 750px.
+
+|*UX URL Control Panel* 
+|urlControlPanel 
+|String 
+|This is a URL to the control panel for editing DNS at the DNS Provider. This field is optional, and allows a Service Provider whose template isn't supported at the DNS Provider to provide a direct link to perform manual edits.

```

Comment:
I'd add here a specification of possible variables.
Similar to other places I would allow for use of %domain% in order to offer a deep link on a certain domain (if supported by the DNS provider)

Reply:
Good idea.